hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -157,7 +157,9 @@ public boolean start(ContainerBalancerConfiguration balancerConfiguration) {
 
       balancerRunning = true;
```

Comment:
I don't think this should be set if config validation fails.

Reply:
@adoroszlai Thanks for the review. Change to set the value after validation.